src/main/resources/changelog/changes/v1.0.0/create-tags-table.xml

```diff
@@ -0,0 +1,40 @@
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+  <changeSet id="task-001@@create-tags-table" author="sherzod">
+    <createTable tableName="tags">
+      <column name="id" type="uuid">
+        <constraints primaryKey="true" primaryKeyName="tags_pkey"
+                     nullable="false"/>
+      </column>
+      <column name="record_id" type="varchar(50)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="record_type" type="varchar(10)">
+        <constraints nullable="false"/>
+      </column>
```

Comment:
Also we should have indexes for record_id + record_type because it will be the most common query and for record_type separate index to provide UI with tags that are available for specific record type

Reply:
done